custodial_copy.tf

```diff
@@ -46,10 +46,10 @@ module "dr2_custodial_copy_db_builder_queue" {
 module "dr2_custodial_copy_queue" {
   source     = "git::https://github.com/nationalarchives/da-terraform-modules//sqs"
   queue_name = local.custodial_copy_name
```

Comment:
I know you didn't add this in this PR but should we rename this slightly? The other one is called "dr2-custodial-copy-queue-creator" but since this is called "dr2-custodial-copy", it might get confused for the other in future, maybe?

Reply:
I see what you mean but the name isn't used for just the queue, it's used for the user and group as well so I'm not sure what we'd change it to.